cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -31,16 +31,20 @@ $$ LANGUAGE plpgsql;
 -- Census-block-level data. TODO: consider renaming the table.
 
 CREATE TABLE IF NOT EXISTS demographics(
-    census_geo_id varchar(255) NOT NULL,
-    census_block_name varchar(255),
-    total_population real,
-    under_five_population real,
-    poverty_population real,
-    black_population real,
-    white_population real,
-    geom GEOMETRY(Geometry, 4326),
-    PRIMARY KEY(census_geo_id)
+   census_geo_id varchar(255) NOT NULL,
+   census_block_name varchar(255),
+   total_population real,
+   under_five_population real,
+   poverty_population real,
+   black_population real,
+   census_state_geo_id varchar(255) NOT NULL references states(census_geo_id),
```

Comment:
Move to an `ALTER TABLE` statement.

Reply:
None of us has a finished table yet. This table will need to be dropped even after this PR to get the zipcode in, unfortunately. I'll be sure to do ALTER TABLE statements once we have finalized tables / are not just writing with lambdas

Reply:
Can you describe that in the PR description then? Otherwise I assume I wouldn't need to drop a table to get a schema update.

Reply:
Added a note about dropping to make aggregations work